rkt/images.go

```diff
@@ -291,7 +336,7 @@ func (f *fetcher) fetchImageFrom(appName string, aciURL, ascURL, scheme string,
 		return "", fmt.Errorf("signature verification for docker images is not supported (try --insecure-skip-verify)")
```

Comment:
I think this is duplicated to https://github.com/coreos/rkt/pull/1353/files#diff-f0cc78e303f77a161e977f01e8374059R276 to https://github.com/coreos/rkt/pull/1353/files#diff-f0cc78e303f77a161e977f01e8374059R282